modules/stochastic_tools/include/distributions/BoostNormalDistribution.h

```diff
@@ -25,5 +25,7 @@ class BoostNormalDistribution : public BoostDistribution<boost::math::normal_dis
   static InputParameters validParams();
 
   BoostNormalDistribution(const InputParameters & parameters);
-};
 
+  Real getMean() const { return _distribution_unique_ptr->mean(); }
```

Comment:
These will need to be removed, because these break builds without Boost. The objects that need can just call getParamTempl<Real>(...). This will also help if the distributions are controllable and we allow for them to change, using a reference to the parameter will capture this change.

Reply:
Funny, I originally was directly getting the input parameters, but I thought this way was cleaner. But it makes sense if we want keep the option of controlling it. Should I do the same thing for the lower and upper bound in UniformDistribution?

Reply:
Yes, it is the same for all the distributions. I went back and forth in my head as well, yesterday and today. Ultimately, just getting the parameter is simpler and will allow controls, as long as the getParam happens up front and isn't called a bunch.